src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -1894,6 +1887,15 @@ bool AdaptDispatch::SetCursorStyle(const DispatchTypes::CursorStyle cursorStyle)
         success = _pConApi->PrivateAllowCursorBlinking(fEnableBlinking);
     }
 
+    // If we're a conpty, always return false, so that this cursor state will be
```

Comment:
Why after not before?

Reply:
I moved this after to make the test work tbh. Doing this after means the conpty will still have this state in it the same as the Terminal, even if there's no way to query or use that information.